fe/fe-core/src/main/java/org/apache/doris/analysis/FromClause.java

```diff
@@ -44,6 +46,8 @@ public class FromClause implements ParseNode, Iterable<TableRef> {
     private boolean analyzed = false;
     private boolean needToSql = false;
 
+    private final Map<Integer, TableRef> tableRefPositions = new HashMap<Integer, TableRef>();
```

Comment:
Could you add comments about the sql caused this bug, and explain why we need this attribute.